homeassistant/components/tessie/sensor.py

```diff
@@ -219,9 +219,8 @@ class TessieSensorEntityDescription(SensorEntityDescription):
     ),
     TessieSensorEntityDescription(
         key="drive_state_active_route_minutes_to_arrival",
-        state_class=SensorStateClass.MEASUREMENT,
-        native_unit_of_measurement=UnitOfTime.MINUTES,
-        device_class=SensorDeviceClass.DURATION,
+        device_class=SensorDeviceClass.TIMESTAMP,
+        value_fn=minutes_to_datetime,
```

Comment:
This data can probably fluctuate a lot. We should use the `ignore_variance` helper. See this example:

https://github.com/home-assistant/core/blob/4c6ac743136a61e6972cad0d76c125b47287e8df/homeassistant/components/prusalink/sensor.py#L176-L179

Reply:
Given this value has floating point accuracy, is it fair to use something less than 2 minutes? Maybe like 15 or 30sec?

I know my usecase for this data is to know as soon as the value falls below 2 minutes, and I just worry a 2 minute variance might kill that.

Reply:
You can pick whatever value you think makes sense for each sensor.

Variance of 2 minutes means that the value needs to change more than 2 minutes from its last reported value for the update to go through. So if the estimated time of arrival would be 2:45:12, it would not update until the value unless it would change to 2:43:12 or lower, or 2:47:12 and higher.